week-3/debug/format-as-12-hours.js

```diff
@@ -1,6 +1,10 @@
 function formatAs12HourClock(time) {
   if (Number(time.slice(0, 2)) > 12) {
-    return `${Number(time.slice(0, 2)) - 12}:00 pm`;
+    let timeFormat = (Number(time.slice(0, 2)) - 12)
+    if (timeFormat <10 )
```

Comment:
Can you think of a way this can be done without an `if` statement? JavaScript's standard library has a great function for this use case 😉